### ISSUE TITLE
Add alt text to all images and logos

### ISSUE THREAD

**Issue by ocastadaniel** [ocastastudios/codebar-git-workshop#11]
Edit index.html and setup.html and groups.html to add sensible alt text to all images and logos in the.

Alt texts should be descriptive to allow people experiencing the website through a screen reader to understand the context of the image.

For example "Ocasta studios logo" rather than "logo"
